prov/efa/src/rxr/rxr_rma.c

```diff
@@ -43,19 +43,19 @@
 #include "rxr_read.h"
 
 int rxr_rma_verified_copy_iov(struct rxr_ep *ep, struct fi_rma_iov *rma,
-			      size_t count, uint32_t flags, struct iovec *iov)
+			      size_t count, uint32_t flags,
+			      struct iovec *iov, void **desc)
 {
 	struct efa_ep *efa_ep;
 	int i, ret;
 
 	efa_ep = container_of(ep->rdm_ep, struct efa_ep, util_ep.ep_fid);
 
 	for (i = 0; i < count; i++) {
-		ret = ofi_mr_verify(&efa_ep->domain->util_domain.mr_map,
-				    rma[i].len,
-				    (uintptr_t *)(&rma[i].addr),
-				    rma[i].key,
-				    flags);
+		ret = ofi_mr_map_verify(&efa_ep->domain->util_domain.mr_map,
```

Comment:
Do you need to lock this call? I know ofi_mr_verify has a lock that is lost with this change.

Reply:
Thanks for catching this. I added the lock back to around ```ofi_mr_map_verify```